### PR TITLE
UI for adding and removing Prime accounts

### PR DESCRIPTION
**Pull request by duy-nguyentuong**

### CODE REVIEW

**Comment by duy-nguyentuong** [comment]
Jul-20-2020 22-15-29
Jul-20-2020 22-16-01

File: ts/prime/Web/src/components/account/PrimeAccountList.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+              })}
+              <PrimeAccountItem isadd={true} onClick={() => history.push('/accounts/prime/add')}>
+                <AddPrimeLogo />
+              </PrimeAccountItem>
+            </PrimeAccountListWrapper>
+          </Spin>
+        </PrimeAccountListBgListener>
+      </BankAccountWrapper>
+      <Modal visible={showConfirm} onOk={handeConfirmDelete} onCancel={() => setShowConfirm(false)}>
+        <Title level={4}>Are you sure you want to delete this Prime account?</Title>
```
Comment (Git-on-my-level) [inline ts/prime/Web/src/components/account/PrimeAccountList.tsx:86]: How about "You are about to remove this Prime account, you can add it back later."

File: ts/prime/Web/src/container/account/AddPrimeAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React, { SyntheticEvent, useEffect, useState } from 'react'
+import AddPrimeAccount from '../../components/account/AddPrimeAccount'
+import { useFormStateAction } from '@/ApiReact'
+import { ActionPrimeAccount } from '@/Api'
+import { message } from 'antd'
+import { useHistory } from 'react-router-dom'
+
+const INVALID_PRIME_NUMBER_ERR = 'No Prime account found with the given number'
```
Comment (Git-on-my-level) [inline ts/prime/Web/src/container/account/AddPrimeAccount.tsx:8]: 'No Prime account found with the given account number'

File: ts/prime/Web/src/components/account/PrimeAccountList.tsx [hunk 3]
```diff
@@ -83,7 +83,7 @@ const PrimeAccountList: React.FC<PrimeAccountListProps> = ({
         </PrimeAccountListBgListener>
       </BankAccountWrapper>
       <Modal visible={showConfirm} onOk={handeConfirmDelete} onCancel={() => setShowConfirm(false)}>
-        <Title level={4}>Are you sure you want to delete this Prime account?</Title>
+        <Title level={4}>You are about to remove this Prime account, you can add it back later?</Title>
```
Comment (Git-on-my-level) [inline ts/prime/Web/src/components/account/PrimeAccountList.tsx:86]: This is not a question!

File: ts/prime/Web/src/components/account/PrimeAccountList.tsx (hunk 1 above)
Comment (Git-on-my-level) [reply ts/prime/Web/src/components/account/PrimeAccountList.tsx]: Check my suggestion again

File: ts/prime/Web/src/components/account/PrimeAccountList.tsx (hunk 3 above)
Comment (duy-nguyentuong) [reply ts/prime/Web/src/components/account/PrimeAccountList.tsx]: Updated

**Pull request merged**